src/test/java/edu/hm/hafner/analysis/assertj/IssueAssert.java

```diff
@@ -0,0 +1,221 @@
+package edu.hm.hafner.analysis.assertj;
+
+import org.assertj.core.api.AbstractAssert;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Priority;
+
+/**
+ * A custom AssertJ assertion to make assertions specific to {@link Issue}.
+ *
+ * @author Andreas Moser
+ */
+public class IssueAssert extends AbstractAssert<IssueAssert, Issue> {
+    /**
+     * Creates a new @link {@link IssueAssert}.
+     *
+     * @param actual The Issue to make assertions on.
+     */
+    public IssueAssert(final Issue actual) {
+        super(actual, IssueAssert.class);
+    }
+
+    /**
+     * An entry point for IssueAssert.
+     *
+     * @param actual The Issue to make assertions on.
+     * @return A new @link {@link IssueAssert}
+     */
+    public static IssueAssert assertThat(final Issue actual) {
+        return new IssueAssert(actual);
+    }
+
+    /**
+     * Verifies that the actual Issue's lineStart is equal to the given one.
+     *
+     * @param line The given lineStart to compare the actual Issue's lineStart to.
+     * @return this assertion object.
+     * @throws AssertionError - if the actual Issue's lineStart is not equal to the given one.
+     */
+    public IssueAssert hasLineStart(final int line) {
+        isNotNull();
+
+        if (actual.getLineStart() != line) {
+            failWithMessage("Expected issue's line start to be <%s> but was <%s>", line, actual.getLineStart());
```

Comment:
Der Standard ist:
`private static final String EXPECTED_BUT_WAS_MESSAGE = "%nExpecting:%n <%s>%nto be equal to:%n <%s>%nbut was not.";`